src/controllers/index.js

```diff
@@ -0,0 +1,3 @@
+const newsRouter = require('./news');
+
+module.exports = [newsRouter];
```

Comment:
any reason for exporting it this way? isn't `module.exports = { newsRouter };` more convenient?